### PR TITLE
Belarusian: add voiced-devoiced rules and test [ready to review]

### PR DESCRIPTION
**Pull request by bespsm**
- add voiced-devoiced rules and test

### CODE REVIEW

File: dictsource/be_rules [hunk 1]
```diff
@@ -75,6 +78,7 @@
 
 .group к
        к       k
+       к (L03  Q
```
Comment (HaraldZealot) [inline dictsource/be_rules:81]: а вось і непраўда, фрыкатыўнасць/выбуховасць няможа змяніцца, таму тут будзе выбухное g

File: tests/language-pronunciation.test [hunk 2]
```diff
@@ -36,7 +40,11 @@ b'apka p;'inxpanQ;ist 'atxadz;its; dz.'Orts.tawn m;'is.ts.as v;'Ests;i" \
 Юрась ад'ютант сальюцца ведаю здолею ёю пенсію мною белую памыю ідэю апошнюю змяю,
 якую з'явіўся гэтая неяк якія пояс халуям дыпламатыя ідэя засмяяўся,
 азноб блог сад ксёндз грандж ажыятаж указ,
-бабка пінгпангіст адхадзіць джорджтаўн міжчас везці" "Cyrl"
+бабка пінгпангіст адхадзіць джорджтаўн міжчас везці,
+апдэйт апгрэйд трохбаковы бухгалтар тэхдогляд трохжыльны тэхзона,
+блокбастар бэкграўнд анекдот экзэма шматбаковы артгрупа бортжурнал спіртзавод,
+звышбуйны Ашгабат звышдоўгі звышжорсткі звышзорка айсберг лясгас гасдагавор сельгасзварот касьба,
```
Comment (HaraldZealot) [inline tests/language-pronunciation.test:46]: And will probably not use soviet acronyms and foreign names as an examples, whenever it is possible

File: dictsource/be_rules (hunk 1 above)
Comment (bespsm) [reply dictsource/be_rules]: fixed

File: tests/language-pronunciation.test (hunk 2 above)
Comment (bespsm) [reply tests/language-pronunciation.test]: fixed

**Comment by HaraldZealot** [comment]
LGTM

**Comment by bespsm** [comment]
please review @jaacoppi  @alex19EP @kirill-jjj

**Comment by bespsm** [comment]
please review PL, it's been waiting for a while @jaacoppi @alex19EP @kirill-jjj

**Pull request merged**